client/state/rewind/retention/reducer.ts

```diff
@@ -0,0 +1,26 @@
+import { AnyAction } from 'redux';
+import {
+	JETPACK_BACKUP_RETENTION_UPDATE,
+	JETPACK_BACKUP_RETENTION_UPDATE_ERROR,
+	JETPACK_BACKUP_RETENTION_UPDATE_SUCCESS,
+} from 'calypso/state/action-types';
+import { combineReducers } from 'calypso/state/utils';
+import { AppState } from 'calypso/types';
+
+const updateBackupRetentionRequestStatus = ( state: AppState = null, { type }: AnyAction ) => {
+	switch ( type ) {
+		case JETPACK_BACKUP_RETENTION_UPDATE:
+			return 'pending';
+
+		case JETPACK_BACKUP_RETENTION_UPDATE_SUCCESS:
+			return 'success';
+
+		case JETPACK_BACKUP_RETENTION_UPDATE_ERROR:
+			return 'failed';
+	}
+	return state;
+};
+
+export default combineReducers( {
+	updateBackupRetentionRequestStatus,
+} );
```

Comment:
@tyxla Could you give some detail on what you mean by this? I think you are referring to getting rid of `combineReducers` since `updateBackupRetentionRequestStatus` is the only reducer currently present. If we do, how can I declare a single reducer?

Reply:
If a reducer has a single subtree, you don't need that it to be defined as a subtree and can just use the parent reducer. That means, exporting the reducer is just fine without a `combineReducers` call. Just like it's done here:

https://github.com/Automattic/wp-calypso/blob/1d652f90440b0e5b0877e9d96cf876518fae7a78/client/state/concierge/is-user-blocked/reducer.js